atlasdb-cassandra-integration-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServiceIntegrationTest.java

```diff
@@ -96,7 +94,12 @@
     private static final Cell CELL = Cell.create(PtBytes.toBytes("row"), PtBytes.toBytes("column"));
     private static final String ASYNC = "async";
     private static final String SYNC = "sync";
+    private static final String CASSANDRA_DEFAULT_TABLE_NAME = "ns__default_table";
+    private static final String ATLAS_DEFAULT_TABLE_NAME = "ns.default_table";
     private final String name;
+    private static final byte[] DEFAULT_TABLE_METADATA = {10, 18, 10, 14, 10, 4, 110, 97, 109, 101, 16, 4, 24, 1, 32, 1,
+            48, 1, 24, 0, 18, 30, 18, 28, 10, 18, 10, 14, 10, 4, 110, 97, 109, 101, 16, 4, 24, 1, 32, 1, 48, 1, 24, 0,
+            18, 6, 8, 4, 24, 1, 32, 3, 24, 2, 32, 64, 48, 0, 64, 0, 72, 1, 88, 0, 96, 0};
```

Comment:
I'm not entirely sure how important testing the Atlas-side table-metadata would be, as that's just stored in the `_metadata` table (so I would imagine this should already be covered by tests of whether we can read or write from Atlas).

Reply:
Discussed offline and removed.